react/features/device-selection/functions.js

```diff
@@ -29,3 +42,143 @@ export function getDeviceSelectionDialogProps(stateful: Object | Function) {
         selectedVideoInputId: settings.cameraDeviceId
     };
 }
+
+/**
+ * Processes device requests from external applications.
+ *
+ * @param {Dispatch} dispatch - The redux {@code dispatch} function.
+ * @param {Function} getState - The redux function that gets/retrieves the redux
+ * state.
+ * @param {Object} request - The request to be processed.
+ * @param {Function} responseCallback - The callback that will send the
+ * response.
+ * @returns {boolean}
+ */
+export function processRequest( // eslint-disable-line max-params
```

Comment:
Could we at least rename this to specify it's processing a request from the external api?